core/org.wso2.carbon.registry.core/src/main/java/org/wso2/carbon/registry/core/jdbc/handlers/XSLTBasedUIEnabledHandler.java

```diff
@@ -56,11 +56,13 @@ public class XSLTBasedUIEnabledHandler extends UIEnabledHandler {
     protected List<String> editViews = new ArrayList<String>();
     protected List<String> newViews = new ArrayList<String>();
 
-    private TransformerFactory transformerFactory;
+    private TransformerFactory transformerFactory = TransformerFactory.newInstance();
```

Comment:
We call a "getSecureTransformerFactory" in the constructor and replace this object. Is this the expected behaviour? 

Or shouldn't we just move the lines 66 to the top of the constructor code?